torch/csrc/jit/passes/loop_unrolling.h

```diff
@@ -7,6 +7,9 @@ namespace jit {
 
 TORCH_API void UnrollLoops(std::shared_ptr<Graph>& graph);
 
+// Only unrolls constant loops. Will unroll them regardless of loop block size
```

Comment:
When `constant_only` is true, UnrollLoop seems to unroll both small-block loops and constant loops (regardless of sizes). The comment states "Only unrolls constant loops", which is confusing.